components/Auction/Auction.module.css

```diff
@@ -1,153 +1,167 @@
-.auctionPageNavigation {
-  text-align: center;
+.newAuctionBtnContainer {
+    text-align: center;
+}
+
+.newAuctionBtn {
+    font-weight: 550;
```

Comment:
550 seems a bit odd to me, I have never seen it being used anywhere. I've generally seen 100,400,800 values. Are you sure this is not a mistake? Of course, you can be right, just that you should be able to back/support your approach.

Reply:
Yes, the number may look odd but i did that in order to make the button look ideal as compared to other elements. Also because 500 was a little less and 600 in font- weight was a little more.